packages/loot-core/src/shared/util.ts

```diff
@@ -269,6 +269,49 @@ export function getNumberFormat({
   };
 }
 
+function getNumberFormatNoDecimal(decimalNumber: number) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏this util looks super similar to `getNumberFormat`. Can we de-dupe this logic?